### PR TITLE
update base infra image to latest (2.2.1)

### PR DESCRIPTION
**Pull request by arvdias**

### CODE REVIEW

**Comment by arvdias** [comment]
Note: caos team may also be updating with the new `nrjmx` version, so we might need to another small version bump in the base image

**Review by gsanchezgavier — DISMISSED** [review]
LGTM thanks!

**Review by roobre — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,16 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
 
+---
+
+## 2.1.0
+
+### Changed
+
+- The base image of `newrelic/infrastructure-k8s` has been updated to `2.2.1`.
```
Comment (roobre) [inline CHANGELOG.md:16]: `2.2.1` or `2.1.0`?

Comment (arvdias) [reply CHANGELOG.md:16]: 2.2.1

Comment (arvdias) [reply CHANGELOG.md:16]: ok, wait I see what you mean.

**Review by gsanchezgavier — APPROVED** [review]

**Pull request merged**